app/src/main/java/com/willowtree/vocable/presets/PresetsFragment.kt

```diff
@@ -110,6 +133,13 @@ class PresetsFragment : BaseFragment<FragmentPresetsBinding>() {
         binding.categoryView.registerOnPageChangeCallback(object :
             ViewPager2.OnPageChangeCallback() {
             override fun onPageSelected(position: Int) {
+
+                binding.phrasesPageNumber.text = getString(
+                        R.string.phrases_page_number,
+                        1,
+                        1
+                    )
```

Comment:
Having this here resolves the issue of the "Page 1 of 1" not appearing for the Recents and 123 category.